objects/b765cf/script.lua

```diff
@@ -144,13 +144,25 @@ function PostSetup(params)
 end
 
 function englandSnap(aidBoard)
-  local snapPoints = aidBoard.getSnapPoints()
-  local newSnapPoints = {}
-  for i,v in ipairs(snapPoints) do
-    if v.rotation.y <= 269 or v.rotation.y >= 271 then
-      table.insert(newSnapPoints,v)
+    do -- Remove aidBoard snap point
+        local snapPoints = aidBoard.getSnapPoints()
+        local newSnapPoints = {}
+        for i,v in ipairs(snapPoints) do
+            if table.concat(v.tags, "|") ~= "Invader Card" then
```

Comment:
the snap point needs to have the invader card tag right? i'm not seeing that change in this PR

Reply:
Yeah, I missed that when originally rebasing, but caught it after pushing.